tests/Fixtures/App/config/bundles.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
```

Comment:
please use the Sonata project header here instead of the Symfony CMF one

Reply:
The right header can be found here:
https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/blob/03b03959b7304c86582f3faf7ec8504b2e468d2b/.php_cs.dist#L10-L15